tests/label/test_label_video_segment.py

```
@@ -147,6 +149,14 @@ def test_extract_and_delete_segment_frame_files(self):
         if sample_frame:
             sample_frame.refresh_from_db()
             self.assertFalse(sample_frame.file_path.exists(), f"Frame file {sample_frame.file_path} should NOT exist after deletion")
+    
+    def test_create_segment_with_video_seg_label_name(self):
+        data = {"video_id": v.id, "label_name": "appendix",
```

Ruff (0.11.9)

Undefined name `v`

(F821)